server/basic_server.js

```diff
@@ -1,10 +1,14 @@
 var express = require('express')
 var app = express()
+var bookshelf = require('./config.js')
 
 app.get('/', function (req, res) {
-  res.send('Hello Brandy Bucks :)!')
+  res.sendFile('Hello Brandy Bucks :)!')
```

Comment:
this is making a conflict in the merge and I think the right thing to do is get rid of it entirely. Using express.static makes this get listener not relevant and confusing. I'll take it out as part of the pull request merge, is that cool Isaac? Also want to try to "Start a review". Pls let me know